### ISSUE TITLE
About the environment

### ISSUE THREAD

**Issue by kira5511** [minar09/LIP-JPPNet-TensorFlow#2]
I need LIP-JPPNet to handle custom images with cp-vton-plus, but I can't generate segmentation images well.

An image like this will be generated
![test](https://user-images.githubusercontent.com/74698192/99624709-0e54aa00-2a72-11eb-8034-80f914107717.jpg)
![test_vis](https://user-images.githubusercontent.com/74698192/99624740-1d3b5c80-2a72-11eb-8f0d-a8fa9c127f31.png)

I made inferences in the following environment, is there any problem?
![aaaaaaaaaaaaaa](https://user-images.githubusercontent.com/74698192/99624620-eb29fa80-2a71-11eb-827b-2a255e605245.PNG)

**Comment by minar09** [comment]
If you can generate the result, I am not sure if the environment has something to do with changing anything. Maybe you can debug your input/output more closely with the intermediate steps.

**Comment by kira5511** [comment]
Thank you for answering　@minar09 

I was seeing some warnings on the console.
If you know the solution from now on, please let me know.

C:\Users\ishikawa_2070\anaconda3\envs\jpp\lib\site-packages\tensorflow\python\framework\dtypes.py:516: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint8 = np.dtype([("qint8", np.int8, 1)])
C:\Users\ishikawa_2070\anaconda3\envs\jpp\lib\site-packages\tensorflow\python\framework\dtypes.py:517: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_quint8 = np.dtype([("quint8", np.uint8, 1)])
C:\Users\ishikawa_2070\anaconda3\envs\jpp\lib\site-packages\tensorflow\python\framework\dtypes.py:518: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint16 = np.dtype([("qint16", np.int16, 1)])
C:\Users\ishikawa_2070\anaconda3\envs\jpp\lib\site-packages\tensorflow\python\framework\dtypes.py:519: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_quint16 = np.dtype([("quint16", np.uint16, 1)])
C:\Users\ishikawa_2070\anaconda3\envs\jpp\lib\site-packages\tensorflow\python\framework\dtypes.py:520: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint32 = np.dtype([("qint32", np.int32, 1)])
C:\Users\ishikawa_2070\anaconda3\envs\jpp\lib\site-packages\tensorflow\python\framework\dtypes.py:525: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  np_resource = np.dtype([("resource", np.ubyte, 1)])
C:\Users\ishikawa_2070\anaconda3\envs\jpp\lib\site-packages\tensorboard\compat\tensorflow_stub\dtypes.py:541: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint8 = np.dtype([("qint8", np.int8, 1)])
C:\Users\ishikawa_2070\anaconda3\envs\jpp\lib\site-packages\tensorboard\compat\tensorflow_stub\dtypes.py:542: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_quint8 = np.dtype([("quint8", np.uint8, 1)])
C:\Users\ishikawa_2070\anaconda3\envs\jpp\lib\site-packages\tensorboard\compat\tensorflow_stub\dtypes.py:543: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint16 = np.dtype([("qint16", np.int16, 1)])
C:\Users\ishikawa_2070\anaconda3\envs\jpp\lib\site-packages\tensorboard\compat\tensorflow_stub\dtypes.py:544: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_quint16 = np.dtype([("quint16", np.uint16, 1)])
C:\Users\ishikawa_2070\anaconda3\envs\jpp\lib\site-packages\tensorboard\compat\tensorflow_stub\dtypes.py:545: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint32 = np.dtype([("qint32", np.int32, 1)])
C:\Users\ishikawa_2070\anaconda3\envs\jpp\lib\site-packages\tensorboard\compat\tensorflow_stub\dtypes.py:550: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  np_resource = np.dtype([("resource", np.ubyte, 1)])
WARNING:tensorflow:
The TensorFlow contrib module will not be included in TensorFlow 2.0.
For more information, please see:
  * https://github.com/tensorflow/community/blob/master/rfcs/20180907-contrib-sunset.md
  * https://github.com/tensorflow/addons
  * https://github.com/tensorflow/io (for I/O related ops)
If you depend on functionality not listed there, please file an issue.

WARNING:tensorflow:From C:\Users\ishikawa_2070\Documents\LIP-JPPNet-TensorFlow\utils\image_reader.py:160: slice_input_producer (from tensorflow.python.training.input) is deprecated and will be removed in a future version.
Instructions for updating:
Queue-based input pipelines have been replaced by `tf.data`. Use `tf.data.Dataset.from_tensor_slices(tuple(tensor_list)).shuffle(tf.shape(input_tensor, out_type=tf.int64)[0]).repeat(num_epochs)`. If `shuffle=False`, omit the `.shuffle(...)`.
WARNING:tensorflow:From C:\Users\ishikawa_2070\anaconda3\envs\jpp\lib\site-packages\tensorflow\python\training\input.py:374: range_input_producer (from tensorflow.python.training.input) is deprecated and will be removed in a future version.
Instructions for updating:
Queue-based input pipelines have been replaced by `tf.data`. Use `tf.data.Dataset.range(limit).shuffle(limit).repeat(num_epochs)`. If `shuffle=False`, omit the `.shuffle(...)`.
WARNING:tensorflow:From C:\Users\ishikawa_2070\anaconda3\envs\jpp\lib\site-packages\tensorflow\python\training\input.py:320: input_producer (from tensorflow.python.training.input) is deprecated and will be removed in a future version.
Instructions for updating:
Queue-based input pipelines have been replaced by `tf.data`. Use `tf.data.Dataset.from_tensor_slices(input_tensor).shuffle(tf.shape(input_tensor, out_type=tf.int64)[0]).repeat(num_epochs)`. If `shuffle=False`, omit the `.shuffle(...)`.
WARNING:tensorflow:From C:\Users\ishikawa_2070\anaconda3\envs\jpp\lib\site-packages\tensorflow\python\training\input.py:190: limit_epochs (from tensorflow.python.training.input) is deprecated and will be removed in a future version.
Instructions for updating:
Queue-based input pipelines have been replaced by `tf.data`. Use `tf.data.Dataset.from_tensors(tensor).repeat(num_epochs)`.
WARNING:tensorflow:From C:\Users\ishikawa_2070\anaconda3\envs\jpp\lib\site-packages\tensorflow\python\training\input.py:199: QueueRunner.__init__ (from tensorflow.python.training.queue_runner_impl) is deprecated and will be removed in a future version.
Instructions for updating:
To construct input pipelines, use the `tf.data` module.
WARNING:tensorflow:From C:\Users\ishikawa_2070\anaconda3\envs\jpp\lib\site-packages\tensorflow\python\training\input.py:199: add_queue_runner (from tensorflow.python.training.queue_runner_impl) is deprecated and will be removed in a future version.
Instructions for updating:
To construct input pipelines, use the `tf.data` module.
WARNING:tensorflow:From C:\Users\ishikawa_2070\Documents\LIP-JPPNet-TensorFlow\utils\image_reader.py:123: The name tf.read_file is deprecated. Please use tf.io.read_file instead.

WARNING:tensorflow:From evaluate_parsing_JPPNet-s2.py:46: The name tf.image.resize_images is deprecated. Please use tf.image.resize instead.

WARNING:tensorflow:From evaluate_parsing_JPPNet-s2.py:53: The name tf.variable_scope is deprecated. Please use tf.compat.v1.variable_scope instead.

WARNING:tensorflow:From C:\Users\ishikawa_2070\Documents\LIP-JPPNet-TensorFlow\kaffe\tensorflow\network.py:45: The name tf.placeholder_with_default is deprecated. Please use tf.compat.v1.placeholder_with_default instead.

WARNING:tensorflow:From C:\Users\ishikawa_2070\Documents\LIP-JPPNet-TensorFlow\kaffe\tensorflow\network.py:99: The name tf.get_variable is deprecated. Please use tf.compat.v1.get_variable instead.

WARNING:tensorflow:From C:\Users\ishikawa_2070\anaconda3\envs\jpp\lib\site-packages\tensorflow\python\ops\init_ops.py:1251: calling VarianceScaling.__init__ (from tensorflow.python.ops.init_ops) with dtype is deprecated and will be removed in a future version.
Instructions for updating:
Call initializer instance with the dtype argument instead of passing it to the constructor
WARNING:tensorflow:From C:\Users\ishikawa_2070\Documents\LIP-JPPNet-TensorFlow\kaffe\tensorflow\network.py:205: The name tf.nn.max_pool is deprecated. Please use tf.nn.max_pool2d instead.

WARNING:tensorflow:From C:\Users\ishikawa_2070\anaconda3\envs\jpp\lib\site-packages\tensorflow\python\util\dispatch.py:180: calling expand_dims (from tensorflow.python.ops.array_ops) with dim is deprecated and will be removed in a future version.
Instructions for updating:
Use the `axis` argument instead
WARNING:tensorflow:From evaluate_parsing_JPPNet-s2.py:136: calling argmax (from tensorflow.python.ops.math_ops) with dimension is deprecated and will be removed in a future version.
Instructions for updating:
Use the `axis` argument instead
WARNING:tensorflow:From evaluate_parsing_JPPNet-s2.py:140: The name tf.global_variables is deprecated. Please use tf.compat.v1.global_variables instead.

 [!] Load failed...
WARNING:tensorflow:From evaluate_parsing_JPPNet-s2.py:159: start_queue_runners (from tensorflow.python.training.queue_runner_impl) is deprecated and will be removed in a future version.
Instructions for updating:
To construct input pipelines, use the `tf.data` module.
step 0
datasets/test/images/test.jpg

Thank you for sparing your precious time for me

**Comment by minar09** [comment]
Hi @kira5511 , it seems your pretrained model loading is failed, that's why the predictions are incorrect. Can you please check whether you downloaded the pretrained model file into the correct path as in your code? Thanks.

`[!] Load failed...`

**Comment by kira5511** [comment]
Hello @minar09 

[!] Load failed... was overlooked.
Changed the path to the correct one and it worked fine
Thank you so much!

**Comment by anantaarora** [comment]
Hi,
The model loading is taking a lot of time. Any idea about how to make the process faster? Please help

**Comment by minar09** [comment]
Hi, @anantaarora , are you running on CPU? Using GPU would make faster maybe. Or you can refer to the original repository https://github.com/Engineering-Course/LIP_JPPNet. Thanks.
